async_producer.go

```diff
@@ -913,6 +913,7 @@ func (bp *brokerProducer) handleSuccess(sent *produceSet, response *ProduceRespo
 			}
 			for i, msg := range pSet.msgs {
 				msg.Offset = block.Offset + int64(i)
+				Logger.Printf("successfully wrote message onto topic %v on partition %d with offset %d", msg.Topic, msg.Partition, msg.Offset)
```

Comment:
if `telegraf` is not set to `debug = True`, I'm guessing this wont log in that case, correct?

Reply:
Also I thought `telegraf`'s kafka output uses a `sync_producer` and not an async one. Is the same change needed there or somehow all roads lead here for sending?

Reply:
Yea `debug = False` means none of these lib logs will log. There is no log level in their logging.

And yes, it does use the `sync_producer` which in turn calls the functionality of the `async_producer`